Rubberduck.VBEEditor/SafeComWrappers/VB/VBComponentFactory.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using Rubberduck.VBEditor.SafeComWrappers.VB.Abstract;
+using VBA = Rubberduck.VBEditor.SafeComWrappers.VB.VBA;
+using VB6 = Rubberduck.VBEditor.SafeComWrappers.VB.VB6;
+using VBAIA = Microsoft.Vbe.Interop;
+using VB6IA = Microsoft.VB6.Interop.VBIDE;
+
+namespace Rubberduck.VBEditor.SafeComWrappers.VB
+{
+    public static class VBComponentFactory
+    {
+        public static IVBComponent Create(object vbComponent)
+        {
+            switch (vbComponent)
+            {
+                case VBAIA.VBComponent _:
```

Comment:
what's that underscore for?

Reply:
switching by type is a C#7 feature, but it requires a variable assignment. The underscore is 'meh, don't care about the assign'